addons/account/tests/test_account_move_entry.py

```diff
@@ -530,6 +533,54 @@ def test_journal_sequence_ordering(self):
         self.assertEqual(copies[5].name, 'XMISC/2019/10005')
         self.assertEqual(copies[5].state, 'draft')
 
+    def test_sequence_concurency(self):
+        with db_connect(self.env.registry.db_name).cursor() as cr0,\
+                db_connect(self.env.registry.db_name).cursor() as cr1,\
+                db_connect(self.env.registry.db_name).cursor() as cr2:
```

Comment:
There is a method `cursor()` on the registry, there is no need for `db_connect` here:
```suggestion
        with self.env.registry.cursor() as cr0,\
                self.env.registry.cursor() as cr1,\
                self.env.registry.cursor() as cr2:
```